src/spikeinterface/sorters/utils/misc.py

```diff
@@ -62,12 +62,16 @@ def has_nvidia():
     """
     Checks if the machine has nvidia capability.
     """
-    from cuda import cuda
+    
+    try:
+        from cuda import cuda
+    except ModuleNotFoundError as err:
+        raise Exception("This container requires cuda, but the package is not installed. You can install it with:\npip install cuda-python") from err
```

Comment:
```suggestion
        raise Exception("This sorter requires cuda, but the package is not installed. You can install it with:\npip install cuda-python") from err
```
this is really dependent on the sorter, and not on running in a container or not.

Reply:
also, `cuda-python` is only used to check if the GPU is available. Maybe you can use this message:
`"The cuda-python package is required to check if a GPU is available. You can install it with:\npip install cuda-python"`

Reply:
Yes but it's only checked if the sorter uses cuda, so is it important for the user to go into the details of exactly why this package is required?

Reply:
then I would do `requires cuda-python`